Makefile

```diff
@@ -409,19 +409,19 @@ mysql-cli:
 test-e2e:
 	# Run E2E tests
 	@mkdir -p test-results
-	go test -timeout 15m -v -count 1 --tags e2e -p 1 --short ./test/e2e 2>&1 | tee test-results/test.out
```

Comment:
these are also enforced by `ci-build.yaml` - don't really need this point of failure